File: codeforces/Next Round.js
var input = readline().split(' ');
var n = input[0], k = input[1];
var cnt = 0;

var arr = readline().split(' ');

for (var i = 0; i < n; i++){
    if (parseInt(arr[i], 10) >= parseInt(arr[k - 1], 10) && parseInt(arr[i], 10) > 0){
        cnt++;
    }
}

print(cnt);

File: codeforces/Next Round.c
#include <stdio.h>

int main(){
    int n, k, cnt = 0;
    scanf("%d %d", &n, &k);
    
    int arr[n];
    for (int i = 0; i < n; i++){
        scanf("%d", &arr[i]);
    }
    
    for (int j = 0; j < n; j++){
        if (arr[j] >= arr[k - 1] && arr[j] > 0){
            cnt++;
        }
    }
    
    printf("%d\n", cnt);
    return 0;
}